sdk/python/tests/utils/test_wrapper_utils.py

```diff
@@ -0,0 +1,14 @@
+import pytest
```

Comment:
[not for this file] I think you can delete `online_write_benchmark.py`? doesn't seem like it's being used anywhere

Reply:
yeah i was wondering about that, i know it's not being used but it had a script, maybe @achals and @adchia can weigh in if this is actually needed